### ISSUE TITLE
Add case document tests to legal/search endpoint

### ISSUE THREAD

**Issue by cnlucas** [fecgov/openFEC#6009]
### What we’re after

As an action item for the postmortem on 10/7, we should increase logging and test coverage for our legal search endpoint. Specifically we should have at least one test that will fail if any of the documents types are not returning which includes both current and archived murs

[Codecov run of legal.py](https://app.codecov.io/gh/fecgov/openFEC/blob/develop/webservices%2Fresources%2Flegal.py)
### Related ticket(s)

(Include the tickets that either came before, after, or are happening in tandem with this new ticket)

- [ ]

### Action item(s)

(These are the smaller tasks that should happen in order to complete this work)

- [ ] [Our test coverage](https://app.codecov.io/gh/fecgov/openFEC/blob/develop/webservices%2Fresources%2Flegal.py) has improved for legal.py
- [ ] A test has been included that would have caught the missing archived murs before deployment

### Completion criteria

(What does the end state look like - as long as this task(s) is done, this work is complete)

- [ ]

### References/resources/technical considerations

(Is there sample code or a screenshot you can include to highlight a particular issue? Here is where you reinforce why this work is important)

**Comment by tmpayton** [comment]
Moving this ticket to the next sprint. This is blocked by the research ticket https://github.com/fecgov/openFEC/issues/6017
